uninstall.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Uninstall script for SM Plugin.
+ *
+ * This file is called when the plugin is uninstalled
+ */
+
+// Exit if accessed directly.
+if (!defined('WP_UNINSTALL_PLUGIN')) {
+    exit;
+}
+
+// Function to remove plugin-specific options from the options table.
+function sm_remove_options() {
+    global $wpdb;
+
+    // Specify the options to be removed.
+    $options = [
+        'sm_post_connector_token',
+        'sm_post_connector_default_post_type',
+        'sm_post_connector_default_author',
+        'sm_post_connector_default_category',
+        'sm_post_connector_secret_key',
+        'sm_post_connector_logo',
+        '_transient_timeout_sm_post_connector_latest_release',
+        '_site_transient_update_plugins',
+    ];
+
+    // Loop through each option and delete it.
+    foreach ($options as $option) {
+        // Use delete_option() for regular options
+        delete_option($option);
+
+        // Use $wpdb->query() to delete transient options, as they might be stored in a different manner
+        $wpdb->query($wpdb->prepare("DELETE FROM {$wpdb->options} WHERE option_name = %s", $option));
+    }
+}
+
+// Function to remove tables.
+function sm_remove_tables() {
```

Comment:
We don't have add any tables, it doesn't make any sense. We just remove settings under options table

Reply:
@rajanvijayan It's removing the fields only not the table

Reply:
@rajanvijayan Please review the changes